pkg/web/leaderboard.go

```diff
@@ -0,0 +1,61 @@
+package web
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/nint8835/scribe/pkg/database"
+	"github.com/nint8835/scribe/pkg/web/ui/pages"
+)
+
+const QUOTES_PER_PAGE = 10
+
+func (s *Server) handleGetLeaderboard(w http.ResponseWriter, r *http.Request) {
+	page := 1
+	if pageStr := r.URL.Query().Get("page"); pageStr != "" {
+		if pageNum, err := strconv.Atoi(pageStr); err == nil {
+			page = pageNum
+		}
+	}
+
+	var quotes []database.Quote
+	var total int64
+
+	query := database.Instance.Model(&database.Quote{}).Where("elo != 1000").Order("elo desc")
```

Comment:
Rather than assuming 1000 == unranked, can this filter out quotes that don't have any completed rankings? My understanding is 1000 will be our average point, so it's not out of the question that once enough votes are completed there'll be some quotes that actually have a rank of 1000

Reply:
This way is better, I'll figure out how to do an efficient where clause on that other table and account for that in the query 👍